enterprise/tools/mount_temp_xfs.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
I also wrote https://gist.github.com/tylerwilliams/58800b42524c31b9036dd3bbe03b855c

to test performance of this stuff.